### PR TITLE
Use CSF in Intro to Storybook

### PR DESCRIPTION
**Pull request by tmeasday**
Basically just https://github.com/chromaui/learnstorybook-code/pull/21 but there is some textual changes that I'd like people to look at.

### CODE REVIEW

File: content/intro-to-storybook/react/en/composite-component.md [hunk 1]
```diff
@@ -240,12 +247,11 @@ Create a test file called `src/components/TaskList.test.js`. Here, we’ll build
 import React from 'react';
 import ReactDOM from 'react-dom';
 import TaskList from './TaskList';
-import { withPinnedTasks } from './TaskList.stories';
+import { WithPinnedTasks } from './TaskList.stories';
```
Comment (tmeasday) [inline content/intro-to-storybook/react/en/composite-component.md]: CSF FTW!

File: content/intro-to-storybook/react/en/data.md [hunk 2]
```diff
@@ -156,26 +163,3 @@ storiesOf('TaskList', module)
     type="video/mp4"
   />
 </video>
-
-Similarly, we need to use `PureTaskList` in our Jest test:
```
Comment (tmeasday) [inline content/intro-to-storybook/react/en/data.md]: Don't need this any more because the test renders the story as exported

File: content/intro-to-storybook/react/en/test.md [hunk 3]
```diff
@@ -56,26 +56,6 @@ Add the package as a dependency.
 yarn add storybook-chromatic
 ```
 
-Import Chromatic in your `.storybook/config.js` file.
```
Comment (tmeasday) [inline content/intro-to-storybook/react/en/test.md]: We don't need to do this step any more (this is unrelated to CSF)

File: content/intro-to-storybook/react/en/composite-component.md (hunk 1 above)
Comment (jonniebigodes) [reply content/intro-to-storybook/react/en/composite-component.md]: the documenation is starting to get along real nice. 💯

File: content/intro-to-storybook/react/en/test.md (hunk 3 above)
Comment (jonniebigodes) [reply content/intro-to-storybook/react/en/test.md]: That's already addressed in svelte, the vue and angular will be updated shortly. 👍

File: content/intro-to-storybook/react/en/composite-component.md [hunk 4]
```diff
@@ -228,9 +235,9 @@ Storybook stories paired with manual visual tests and snapshot tests (see above)
 
 However, sometimes the devil is in the details. A test framework that is explicit about those details is needed. Which brings us to unit tests.
 
-In our case, we want our `TaskList` to render any pinned tasks **before** unpinned tasks that it has passed in the `tasks` prop. Although we have a story (`withPinnedTasks`) to test this exact scenario, it can be ambiguous to a human reviewer that if the component **stops** ordering the tasks like this, it is a bug. It certainly won’t scream **“Wrong!”** to the casual eye.
+In our case, we want our `TaskList` to render any pinned tasks **before** unpinned tasks that it has passed in the `tasks` prop. Although we have a story (`WithPinnedTasks`) to test this exact scenario, it can be ambiguous to a human reviewer that if the component **stops** ordering the tasks like this, it is a bug. It certainly won’t scream **“Wrong!”** to the casual eye.
```
[the remark this answers is not in the document]
Comment (tmeasday) [reply content/intro-to-storybook/react/en/composite-component.md]: Sure/

File: content/intro-to-storybook/react/en/simple-component.md [hunk 5]
```diff
@@ -78,17 +89,21 @@ There are two basic levels of organization in Storybook: the component and its c
...
 
-Another nice thing about bundling the `actions` that a component needs is that you can `export` them and use them in stories for components that reuse this component, as we'll see later.
+As we need to pass the same set of actions to all permutations of our component, it is convenient to bundle them up into a single `actionsData` variable and use React's `{...actionsData}` props expansion to pass them all at once. `<Task {...actionsData}>` is equivalent to `<Task onPinTask={actionsData.onPinTask} onArchiveTask={actionsData.onArchiveTask}>`.
 
-To define our stories, we call `add()` once for each of our test states to generate a story. The action story is a function that returns a rendered element (i.e. a component class with a set of props) in a given state---exactly like a React [Stateless Functional Component](https://reactjs.org/docs/components-and-props.html).
+Another nice thing about bundling the `actionsData` that a component needs is that you can `export` them and use them in stories for components that reuse this component, as we'll see later.
```
[the remark this answers is not in the document]
Comment (tmeasday) [reply content/intro-to-storybook/react/en/simple-component.md]: I'll reword it, I'm unsure about adding a comma there.

**Comment by tmeasday** [comment]
I'd like @domyen to take a look before we merge this, although I realise he is pretty busy right now.

**Comment by domyen** [comment]
lgtm

**Comment by jonniebigodes** [comment]
@tmeasday i completely forgot about that...the line feature i mean. It was one of those homer simpson moments (pardon the bad pun). If you can just update the get started section, more specifically the assets part, to match the svelte one, with that we can start making the documentation more uniform, leaving out only the angular one, that i'll update shortly.

**Comment by tmeasday** [comment]
> If you can just update the get started section, more specifically the assets part, to match the svelte one, with that we can start making the documentation more uniform, leaving out only the angular one, that i'll update shortly.

Should we do that separately? Might make it easier to copy this change to other places?

**Comment by jonniebigodes** [comment]
We could do this on a separate pr. The changes already exist in the documentation. It's just a question of making a small corrective pr and we're done. If @domyen is ok with this. I think we can merge this.

**Comment by domyen** [comment]
Sounds good @jonniebigodes! Merge away 🚀

**Comment by jonniebigodes** [comment]
@tmeasday when you're ready and able merge this and we'll continue from here

**Comment by tmeasday** [comment]
🎉

**Comment by tmeasday** [comment]
Thanks @jonniebigodes

**Comment by jonniebigodes** [comment]
No problem, glad i was able to help out
